Sources/PackageModel/Manifest.swift

```diff
@@ -15,12 +15,14 @@ import Utility
 public enum ManifestVersion: String, Codable {
     case v4 = "4"
     case v4_2 = "4_2"
+    case v5 = "5"
```

Comment:
I think @aciidb0mb3r will make a change soon to support v5, this PR should be based on that.

Reply:
indeed, thanks, done